Sources/tart/Commands/Run.swift

```diff
@@ -112,7 +112,18 @@ struct Run: AsyncParsableCommand {
 
   @MainActor
   func run() async throws {
-    let vmDir = try VMStorageLocal().open(name)
+    let localStorage = VMStorageLocal()
+    let vmDir = try localStorage.open(name)
+    let needToGenerateNewMac = try localStorage.list().contains {
```

Comment:
We should probably guard this logic with:

1. VM's `state()` check, to only generate new MAC-addresses for suspended VMs and balance with the same check in `tart clone`.
2. A lock, see https://github.com/cirruslabs/tart/pull/539#discussion_r1254420932.

Reply:
Fixed in [ed4b3e1](https://github.com/cirruslabs/tart/pull/539/commits/ed4b3e16d96196d325bc62bfb28211c30a83e460)